bson/value.go

```diff
@@ -553,6 +553,16 @@ func (v *Value) Binary() (subtype byte, data []byte) {
 	return st, b
 }
 
+// BinaryOK is the same as Binary, except it returns a boolean instead of
+// panicking.
+func (v *Value) BinaryOK() (subtype byte, data []byte, bool) {
```

Comment:
There's a compilation issue here, the `bool` parameter needs a name. Changing this to `ok bool`, should work.

Reply:
Good catch, sorry for the mistake.